### DISCUSSION TITLE
Unable to add dev container configuration files

### DISCUSSION THREAD

**Discussion by anthonychu** [community/community#10552] [Codespaces]
I'm not able to find the `Add Development Container Configuration Files` command in Codespaces anymore. I'm testing this by creating a codespace in a new, empty repo.

<img width="890" alt="image" src="https://user-images.githubusercontent.com/3982077/151054384-85cfa6cd-9207-4fce-b4ec-d077f7ebb951.png">

I can find these Codespaces commands:

<img width="910" alt="image" src="https://user-images.githubusercontent.com/3982077/151054510-02a7fbfe-0023-446a-9ac3-c9aa90430606.png">

But none of them appear to work:

<img width="552" alt="image" src="https://user-images.githubusercontent.com/3982077/151054557-bb53d8c0-26aa-47a4-ac46-56dc1f1d3409.png">

**Comment by joshspicer** [comment]
Hey @anthonychu  👋 - Would you mind trying again? It seems to be ok for me with your test repo, but perhaps there's something else different between our setups (and if that's the case i'd love to dig into that and get it fixed). Thanks!

![image](https://user-images.githubusercontent.com/23246594/151884198-70af5046-0c42-4252-85e4-05472de187f6.png)

**Comment by anthonychu** [comment]
Thanks @joshspicer. I deleted the codespace and created a new one from that repo and still the same error. I think it's something with my settings or something but I can't figure out what it is. 😕 If you can get on a screen share or something, reach out to me at antchu at microsoft. Thanks.

**Reply by joshspicer** [reply]
will reach out!

**Reply by joshspicer** [reply]
For anyone else hitting this @anthonychu and I discovered his version of Edge insiders was not up-to-date, and thus he was hitting a websockets bug! https://github.com/github/feedback/discussions/10129#discussioncomment-2020803
